source/inputCore.py

```diff
@@ -684,17 +696,19 @@ def className(self):
 
 def normalizeGestureIdentifier(identifier):
 	"""Normalize a gesture identifier so that it matches other identifiers for the same gesture.
-	Any items separated by a + sign after the source are considered to be of indeterminate order
-	and are reordered into Python set ordering.
-	Then the entire identifier is converted to lower case.
+	First, the entire identifier is converted to lower case.
+	Then, any items separated by a + sign after the source prefix are considered to be of indeterminate order
+	and are sorted by character.
 	"""
+	identifier = identifier.lower()

```

Comment:
is this locale dependant? Will it use the system locale or the invariant locale by default?